lib/mapper-v1.js

```diff
@@ -34,7 +35,25 @@ exports.identify = function(msg, settings) {
   ret.user_id = msg.userId();
   ret.custom_attributes = traits;
 
-  if (msg.created()) ret.remote_created_at = time(msg.created());
+  var created = time(msg.created())
+
+  if (created) {
+    ret.created_at = created;
+    remove(ret.custom_attributes, 'created_at');
+    remove(ret.custom_attributes, 'createdAt');
+  }
+
+  var remoteCreated = traits.remote_created_at || traits.remoteCreatedAt;
+
+  if (remoteCreated) {
+    var formatted_remote_created = newDate(remoteCreated);
+    ret.remote_created_at = time(formatted_remote_created);
+
+    // delete dupes
+    remove(traits, 'remote_created_at');
```

Comment:
Hey Chris, yes - `created_at` and `remote_created_at` are top-level attributes in Intercom and shouldn't also be passed as custom attributes.